apps/client/src/components/category/category.tsx

```diff
@@ -2,7 +2,7 @@ import { IoMdAddCircleOutline } from 'react-icons/io';
 import styles from './category.module.css';
 import React, { PropsWithChildren, useState } from 'react';
 import { Tooltip } from 'react-tooltip';
-import { AddToast } from '../add-toast/add-toasts';
+import { AddToast } from '..';
```

Comment:
I don't think this is a valid import route

Reply:
why?

Reply:
After asking postol apparently this is valid although weird.
My bad though you can resolve